maxim/tests/test_livekit.py

```
@@ -18,16 +19,17 @@
     def __init__(self) -> None:
-        super().__init__(instructions="You are a helpful voice AI assistant.")
+        super().__init__(
+            instructions="You are a helpful voice AI assistant.",
+            stt=openai.STT(model="gpt-4o-mini-transcribe"),
+            llm=google.LLM(model="gemini-2.0-flash-001", temperature=1),
+            tts=openai.TTS(model="gpt-4o-mini-tts", voice="alloy"),
+            vad=silero.VAD.load(),
+        )
 
 
 async def entrypoint(ctx: agents.JobContext):
```

Ruff (0.12.2)

Missing return type annotation for public function `entrypoint`

Add return type annotation: `None`

(ANN201)